spec/inverted-index-helper.js

```diff
@@ -1,25 +1,55 @@
-class invertedIndexHelper {
+/**
+ * InvertedIndexHelper
```

Comment:
Reasons being that, the `window` object is not available for Jasmine testing and using an `if` condition did no resolve the issue.